unitex/build.xml

```diff
@@ -43,7 +43,7 @@
 
 	<target name="compile" depends="init" description="compile the source">
 		<!-- Compile the java source code -->
-		<javac target="1.7" source="1.7" encoding="UTF-8" srcdir="${src-dir}" destdir="${build-dir}" includeAntRuntime="false">
+		<javac target="1.8" source="1.8" encoding="UTF-8" srcdir="${src-dir}" destdir="${build-dir}" includeAntRuntime="false">
```

Comment:
The target JRE is currently set to1.7. Can you explain why it is necessary to stop to support this version ?

Reply:
I used some lambda expressions in my code that is why I changed It to 1.8 , but I can replace those lambda expressions by anonymous classes and put 1.7 back

Reply:
IMO except allowing to use the new features in Java 8, 1.7 shouldn't be supported anymore since it doesn't get updates since 2015.

Reply:
Using Java 8 as the minimum version would be possible for the next major release, i.e. Unitex/GramLab v4.0. For now,  do not change the target version.